.github/workflows/tests.yml

```diff
@@ -571,6 +571,7 @@ jobs:
         env:
           POSTGRES: ${{ (matrix.database == 'Postgres') && 1 || '' }}
           WORKERS: ${{ (matrix.arrangement == 'workers') && 1 || '' }}
+          UNIX_SOCKETS: 1
         name: Run Complement Tests
 
   cargo-test:
```

Comment:
Seems like this is related to https://github.com/matrix-org/synapse/issues/4975

We should add a `Part of https://github.com/matrix-org/synapse/issues/4975` to the issue description if this is the case (or `Fix ...` if this is the final piece)